CONTRIBUTING.rst

```diff
@@ -125,7 +125,9 @@ Then run (assuming the main fork is in a git remote called `upstream`)::
 $ git checkout master
```

Comment:
Unrelated but, since we're touching this file, would you mind updating the `TODO` above in this doc to the bugzilla link for filing bugs in Glean: SDK?